projects/ngneat/transloco/src/lib/transloco.service.ts

```diff
@@ -280,22 +281,22 @@ export class TranslocoService implements OnDestroy {
     return lang;
   }
 
-  private _setTranslation(lang: string, translation: Translation) {
+  private _setTranslation(lang: string, translation: Translation, scopeAlias?: string) {
     const withHook = this.interceptor.preSaveTranslation(translation, lang);
     this.translations.set(lang, withHook);
     const { scopeStrategy, scopeMapping = {} } = this.config;
```

Comment:
Before I write more comments, why are you passing this property all the way down? You can update `scope.scopeMapping` inside the directive and the pipe. Btw, it should be:

`const { scopeStrategy, scopeMapping = {} } = this.mergedConfig` @shaharkazaz

Reply:
My bad. Didn’t  think about it. Will update the code. Thanks!

Reply:
No problem. V2 is ready. Please, merge your code into the v2 branch and continue the work there.

Reply:
* I meant merge v2 into your branch.

Reply:
@NetanelBasal I've made changes to set alias directly in directive and pipe. 
The only thing is that v2 branch's playground is broken. 'messageformat' package is missing. 
Could you please merge master changes to v2 so I can rebase and fix the issue? or should I add it manually to my branch?

Reply:
Push the code to your branch. I will fix the merging issues.